source/widget.js

```diff
@@ -62,7 +62,7 @@ Likely.prototype = {
     update: function (options) {
         if (
             options.forceUpdate ||
-            options.url !== this.options.url
+            options.url && options.url !== this.options.url
```

Comment:
In another words: `false || false && true => false` (don't forget about priority of the operations, `&&` first and the `||`, so we don't need extra parenthesis)